internal/datastore/datastore.go

```diff
@@ -13,6 +14,13 @@ import (
 
 var Engines = []string{}
 
+// SortedEngineIDs returns the full set of engine IDs, sorted.
+func SortedEngineIDs() []string {
+	engines := append([]string{}, Engines...)
```

Comment:
This will drop the quotes around each engine. Previously it was

> "memory", "postgres" ...

But now it will be

> memory, postgres ...

Nothing blocking, but it will look different from other flags that list valid options.

Reply:
Fixed